### ISSUE TITLE
Reference to other Embedded github content does not always work

### ISSUE THREAD

**Issue by Baasie** [Virtual-Domain-driven-design/virtual-domain-driven-design#24]
At the moment we embed github repo's through the `gatsby-source-git` pluging in the `gatsby-config.js` like:
```{
      resolve: `gatsby-source-git`,
      options: {
        name: `github-repo-ddd-crew-starter-modelling`,
        remote: `https://github.com/ddd-crew/ddd-starter-modelling-process.git`,
      },
    },```

Then on the `gatsyb-node.js` file we created pages for all markdown files in the repo except license and licence:
We link the new MD file to the `./src/templates/github-repo-layout.tsx` that will display those MD files from github.

From here we use the `markdown-link.tsx` component so that we can change the links that are used in the github markdown files.

The problem is, these links are usually relative to the github repo, like /example. This does not work with gatsby because it will create a link to virtualddd.com/example. The behaviour we need is to have the virtualddd.com/{currentpath}/example.

It now works for the current situation, but it needs to be extended for more examples in the future so all links possible will work.

**Comment by yellowbrickc** [comment]
I assume, this issue is partly fixed with #47

**Comment by yellowbrickc** [comment]
The relative links work now, the static images, pdfs, mir backup files too.
